contracts/LanVar.sol

```diff
@@ -1,32 +1,85 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.4;
+pragma solidity ^0.8.12;
 
 import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
 import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
 import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/utils/Counters.sol";
+import "@chainlink/contracts/src/v0.8/ChainlinkClient.sol";
 
-contract LanVar is ERC721, ERC721Enumerable, ERC721URIStorage, Ownable {
+contract LanVar is ERC721, ERC721Enumerable, ERC721URIStorage, Ownable, ChainlinkClient {
     using Counters for Counters.Counter;
+    using Chainlink for Chainlink.Request;
+
+    event MintSuccessful(string tokenURI, address owner);
```

Comment:
I think we can combine these two events into one MintNFTCreated with the following parameters

- string tokenURI
- address owner
- string tokenID
- bool status